src/graphql/data/util.tsx

```diff
@@ -133,7 +133,7 @@ export const CHAIN_NAME_TO_CHAIN_ID: { [key in Chain]: SupportedChainId } = {
 }
 
 export function fromGraphQLChain(chain: Chain): SupportedChainId {
-  return CHAIN_NAME_TO_CHAIN_ID[chain]
+  return CHAIN_NAME_TO_CHAIN_ID[chain] ?? SupportedChainId.MAINNET
```

Comment:
hmm the backend here is still gonna return avax tokens - theyre gonna show up as mainnet instead?

Reply:
the avax tokens won't show up until we modify our queries to add them, at which point we can add them to `CHAIN_NAME_TO_CHAIN_ID` as well

Reply:
defaulting to main is the current behaviour for our gql chain behaviour including portfolio https://github.com/Uniswap/interface/blob/main/src/components/AccountDrawer/MiniPortfolio/PortfolioLogo.tsx#L88